.github/workflows/ci.yml

```diff
@@ -34,6 +34,12 @@ jobs:
             PHP_CS_FIXER_TEST_USE_LEGACY_TOKENIZER: 1
             SYMFONY_DEPRECATIONS_HELPER: 'disabled'
 
+          - operating-system: 'ubuntu-20.04'
+            php-version: '7.4'
+            job-description: 'with calculating code coverage'
+            calculate-code-coverage: 'yes'
+            phpunit-flags: '--testsuite coverage --exclude-group covers-nothing --coverage-clover build/logs/clover.xml'
```

Comment:
That's quite complicated here. What do you think about splitting this part into a dedicated workflow? 

It will be much simpler and easier to maintain this way. A lot less of iffs in code.

Reply:
Good idea will do it after https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/5314 is merged.

Reply:
pls, keep the code coverage in `ci.yaml` workflow

ultimately, most (or even each) job from matrix should report code coverage, not only 7.4 (we done only 7.4 in Travis for sake of speed).
execution path is different on 5.x, 7.0, 7.4 and 8.0.

Reply:
Ok

Reply:
Why can't we have a separate `.yaml` with all PHP versions we want to collect coverage against?

Reply:
because that yaml file would have all PHP versions and other yaml file would be empty

Reply:
The other file will have only 5.x, 7.0, 7.4, and 8.0, 'cause like you point out they have different execution paths.

Reply:
so, if each PHP version execute different lines of code, we should collect code coverage from each of them, don't we?

Reply:
Yes, that's the point. We make another .yaml workflow, that runs only these four versions, and only tests that cover anything at all (e.g. no autoreview tests). And nothing else, _unlike_ in this workflow that not only tests but also dogfoods the program.

Reply:
i'm open to see any working version of this PR before further judgement

Reply:
It is working for a week, I've rebased it to have changed from https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/5314, so I could remove the step with bumping PHPUnit to a version supporting pcov.

Reply:
can we have the coverage reported on PR ?

Reply:
if/when it's ready, please convert to regular PR